both/helpers.js

```diff
@@ -32,13 +37,13 @@ register("competition", function(attribute) {
 });
 
 register("roundEventCode", function() {
-  return getRoundAttribute(this.roundId, 'eventCode');
+  return getRound(this.roundId).eventCode;
```

Comment:
It would probably be more efficient to get rid of these methods entirely. The reason for them being like this was to try to get finer grained reactivity, which I'm largely convinced is not the right way to achieve performance. Not something we need to do in this PR.